src/content/security/index.md

```diff
@@ -0,0 +1,231 @@
+
+<Divider />
+
+## Crypto Security 101
+
+### Upgrade Your Knowledge
+
+One of the biggest reasons people get scammed in crypto generally is a lack of understanding. For example, if you don't understand that the Ethereum network is decentralized and owned by no one, then it's easy to fall prey to someone pretending to be a customer service agent that promises to return your lost ETH in exchange for your private keys. Educating yourself on how Ethereum works is a worthwhile investment.
+
+<DocLink to="/what-is-ethereum/" title="What is Ethereum?" />
+
+<DocLink to="/eth/" title="What is ether?" />
+<Divider />
+## Wallet Security
+
+### Don't give out your Private keys
+
+**Never, for anyone reason, hand out your private keys!**
+
+The private key to your wallet acts as your digital signature. It is the only thing stopping someone who knows your wallet address from draining your account of all of its assets!
+
+### Use a Hardware Wallet
+
+A hardware wallet provides offline storage for private keys. They are considered the most secure wallet option for storing your private keys.
+
+Keeping private keys offline massively reduces the risk of being hacked, even if a hacker gets control of your computer.
+
+#### Try a hardware wallet:
+
+- [Ledger](<[https://www.ledger.com/](https://www.ledger.com/)>)
+- [Trezor](<[https://trezor.io/](https://trezor.io/)>)
+
+### Double check transactions
+
+Accidently sending crypto to the wrong wallet address is a common mistake. Unless you know the address owner, there will be no way for you to retrieve your funds.
+
+Always make sure the address you are sending to exactly matches the desired recipient's address before sending a transaction.
+It is also recommended when interacting with a smart contract to read the transaction message before signing.
+### Set smart contract spend limits
+
+When interacting with smart contracts, do not allow unlimited spend limits. An unlimited spend could enable the smart contract to drain your wallet. Instead, set spending limits to only the amount necessary for the transaction.
+
+### Don't take screenshots of your seed phrases/private keys
+
+By screenshotting your seed phrases or private keys, you risk syncing them to the cloud and potentially making them accessible to hackers. Obtaining private keys from the cloud is a common attack vector for hackers.
+
+<Divider />
+
+## Common Scams
+
+Todo: add copy here
+
+### Giveaway scam (Double your ETH)
+
+One of the most common scams in cryptocurrency is the giveaway scam. The giveaway scam can take many forms, but the general premise is that if you send ETH to the provided wallet address, you will receive your ETH back but doubled. *For this reason, it is also known as the 2-for-1 scam.*
+
+These scams usually stipulate a limited time of opportunity to claim the giveaway to encourage poor decision-making and create a sense of false urgency.
+
+#### Twitter Hack
```

Comment:
Good example. I might broaden this to "social media hacks"? Another pretty common one is for a hacker to hijack a popular YouTube channel (e.g. with millions of subscribers) then use it to start pushing scams.

Reply:
Ok stop it in it's tracks